src/widget/wlabel.cpp

```diff
@@ -43,26 +45,62 @@ void WLabel::setup(QDomNode node, const SkinContext& context) {
     setPalette(pal);
 
     // Text
-    if (context.hasNode(node, "Text"))
-        m_qsText = context.selectString(node, "Text");
-    setText(m_qsText);
+    if (context.hasNodeSelectString(node, "Text", &m_skinText)) {
+        setText(m_skinText);
+    }
 
     // Font size
-    if (context.hasNode(node, "FontSize")) {
-        int fontsize = context.selectString(node, "FontSize").toInt();
+    QString strFontSize;
+    if (context.hasNodeSelectString(node, "FontSize", &strFontSize)) {
+        int fontsize = strFontSize.toInt();
         setFont(QFont("Helvetica", fontsize, QFont::Normal));
```

Comment:
"Helvetica" is just the Qt default 
http://doc-snapshots.qt.io/4.8/qfont.html
This value seems to guarantees that Qt picks the default font as usually. 
I am still unsure about it. But I will drop a comment.